resources/assets/js/slugify.js

```diff
@@ -101,10 +101,9 @@
 
                 str = _slug
                     .replace(/^\s+|\s+$/g, '')      // Trim
-                    .replace(/[^-\u0600-۾\w\d\$\*\(\)\'\!\_]/g, _sep)   // Remove invalid chars
+                    .replace(/[^-\u0600-۾\w]/g, _sep)   // Remove invalid chars
```

Comment:
Is this ok? @fletch3555

Reply:
At quick glance, it seems like it would meet the requirements.  We'll have to do a bit of testing